docs/src/guides/explorer.md

```diff
@@ -71,3 +71,33 @@ Or by percentile, the percentage of bitcoin's supply that has been or will have
 been issued when they are mined:
 
 [100%](https://ordinals.com/search/100%)
```

Comment:
Should remove what I added from above then since this explains it much better `Ln 17-19` or move it down here.
```diff
-To enable the JSON-API endpoints add the `--enable-json-api` or `-e` flag:
-`ord --enable-json-api server`
```

Reply:
I think having that at the top is fine. Let's link to the below paragraph, I'll do that real quick